packages/commerce-sdk-react/src/test-utils.tsx

```diff
@@ -34,6 +35,20 @@ export const DEFAULT_TEST_CONFIG = {
     OCAPISessionsURL: `${DEFAULT_TEST_HOST}/mobify/proxy/ocapi/s/RefArch/dw/shop/v22_8/sessions`
 }
 
+export const API_CLIENTS_KEYS: (keyof ApiClients)[] = [
+    'shopperBaskets',
+    'shopperContexts',
+    'shopperCustomers',
+    'shopperExperience',
+    'shopperGiftCertificates',
+    'shopperLogin',
+    'shopperOrders',
+    'shopperProducts',
+    'shopperPromotions',
+    'shopperSearch',
+    'shopperSeo'
+]
```

Comment:
Is there a way that we don't have to re-declare this list of API clients?

Reply:
Follow up.. if we dont have to keep a running tally of the apis we wouldn't really require [this](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1701/files#diff-a9025efe405e93fdf4a6a0d4bac3966d89a61c4844e17bde31276d3130e2035aR18) check.. I don't think